tests/tt_eager/integration_tests/test_bert.cpp

```diff
@@ -285,7 +285,7 @@ void test_bert() {
     run_loop();
     device->disable_and_clear_program_cache();
 
-    TT_FATAL(tt::tt_metal::CloseDevice(device));
+    TT_FATAL(tt::tt_metal::CloseDevice(device), "Error");
```

Comment:
Does this actually make sense? 

If close device returned false I will see "Error" but not info on some of the typical reasons and how to deal with it.
I find this error message is useless.

See https://github.com/tenstorrent/tt-metal/blob/main/ErrorMessageBestPractices.md

Reply:
TT_FATAL prints out the assert type, file, line, and condition that failed, in this case that the fatal failed on the condition `tt::tt_metal::CloseDevice(device)`.

That should be sufficient, especially since this is for our internal test.